.github/workflows/ci_reusable_wf.yml

```diff
@@ -0,0 +1,349 @@
+name: Reusable Dev CI
```

Comment:
Should't we call this from `ci.yml`?

Reply:
Yes, but in order for this to work, it has to be first on the main branch. So I have to create a PR for creating the file and merging it into main and then I can call it on `ci.yml`.